pkg/nimble/netif/include/nimble_netif.h

```diff
@@ -76,6 +76,15 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   The maximum number of BLE connections that can be open concurrently
+ *
+ * @note    This value *must* be <= @ref MYNEWT_VAL_BLE_MAX_CONNECTIONS
```

Comment:
Since our `doxygen` config has no access to MyNewt's / NimBLE's documentation, this reference goes nowhere, so doxygen fails (see both Travis and Murdock log). Please remove the `@ref`
```suggestion
 * @note    This value *must* be <= MYNEWT_VAL_BLE_MAX_CONNECTIONS
```